app/Domain/Clients/Repositories/Clients.php

```diff
@@ -179,7 +185,7 @@ public function getClientsUsers($clientId): false|array
         /**
          * addClient - add a client and postback test
          */
-        public function addClient(array $values): false|string
+        public function addClient(object $values): false|string
```

Comment:
lastInsertId from below should return an int instead of string. Please change from false|string to false|int